src/generation-logic/generate.js

```diff
@@ -51,14 +54,19 @@ function getRandom(n, data) {
     return arr[0];
 }
 
+function getRandomRoundedPercentage(step) {
+    let rnd = Math.random();
```

Comment:
This can be `const`

Reply:
That's true! Even inline, actually, because its value is used only once (in all cases, not only this one).